WBooks/Models/BookRequest.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  NewBook.swift
+//  WBooks
+//
+//  Created by sebastian.a.spadea on 18/04/2022.
+//
+
+import Foundation
+
+public struct BookRequest: Codable {
```

Comment:
We can use the same Book class for the requests, in this case it is not necessary to create a new model